js/src/api/subscriptions/personal.js

```diff
@@ -37,12 +39,33 @@ export default class Personal {
     ]);
   }
 
+  // FIXME: Because of the different API instances, the "wait for valid changes" approach
+  // doesn't work. Since the defaultAccount is critical to operation, we poll in exactly
+  // same way we do in ../eth (ala same as eth_blockNumber) and update. This should be moved
+  // to pub-sub as it becomes available
   _defaultAccount = () => {
+    const nextTimeout = (timeout = 1000) => {
+      setTimeout(() => {
```

Comment:
Yes ok, so the `_defaultAccount` polling gets cancelled when the API gets nuked, right ? We might want to just keep track of it in case we case we want to stop the polling ; just by adding `this._defaultAccountTimeoutId = setTimeout` would do I guess. Would be easier to then move to a proper logic of stopping the polling when the Transport is disconnected, starting when it's connected ; stopping when there are no subscribers, etc..